cookiecutter.json

```diff
@@ -3,6 +3,7 @@
   "bot_display_name": "AsyncBox bot",
   "bot_short_description": "TODO",
   "bot_docs_link": "TODO",
+  "is_private_dependencies": false,
```

Comment:
Maybe `has_private_dependencies`?  
And one more thing. I suppose you should write about this point in more detail because it seems to me that users will leave this value by default, and then wonder why an error raises while dependencies loading.
Or simply change the default value on `true`.

Reply:
naming - 👍 
the default value - `false` for partners